presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -19,6 +19,29 @@ Presto Geospatial functions support the Well-Known Text (WKT) form of spatial ob
 * ``MULTIPOLYGON (((0 0, 4 0, 4 4, 0 4, 0 0), (1 1, 2 1, 2 2, 1 2, 1 1)), ((-1 -1, -1 -2, -2 -2, -2 -1, -1 -1)))``
 * ``GEOMETRYCOLLECTION (POINT(2 3), LINESTRING (2 3, 3 4))``
 
+Use ST_GeometryFromText and ST_GeomFromBinary functions to create geometry objects from WKT or WKB.
+
+The SphericalGeography type provides native support for spatial features represented on "geographic" coordinates (sometimes called "geodetic" coordinates,
+or "lat/lon", or "lon/lat"). Geographic coordinates are spherical coordinates expressed in angular units (degrees).
+
+The basis for the Geometry type is a plane. The shortest path between two points on the plane is a straight line. That means calculations on geometries (areas, distances, lengths,
+intersections, etc) can be calculated using cartesian mathematics and straight line vectors.
+
+The basis for the SphericalGeography type is a sphere. The shortest path between two points on the sphere is a great circle arc. That means that calculations on geographies
+(areas, distances, lengths, intersections, etc) must be calculated on the sphere, using more complicated mathematics. More accurate measurements that take the actual spheroidal
+shape of the world into account are not supported.
+
+Because the underlying mathematics is much more complicated, there are fewer functions defined for the geography type than for the geometry type. Over time, as new algorithms are
+added, the capabilities of the geography type will expand.
+
+The units returned by the measurement functions (ST_Distance, ST_Length, ST_Area) are in meters.
+
+Use ``to_spherical_geography()`` function to convert a geometry object to geography object.
+
+For example, ``ST_Distance(ST_Point(-71.0882, 42.3607), ST_Point(-74.1197, 40.6976))`` returns 3.4577 in angular units, while
```

Comment:
Talking about "angular unit" is confusing, especially since it's not a very meaningful concept for a system based on latitude and longitude due to the axis being asymmetric (there are 360 degrees of longitude but only 180 degrees of latitude). I don't think we should even give an example that uses the lat/lon in geometry-based distance and area functions.

Reply:
> for a system based on latitude and longitude due to the axis being asymmetric (there are 360 degrees of longitude but only 180 degrees of latitude)

Actually, scratch that. The ranges are not symmetric, but the size of each unit along each axis is the same, so relative values are meaningful. However, it's incorrect to say those functions "return x in angular units". They return a result in the (implicit) unit of the values on the euclidean space that were passed in. So "angular" has no relevance here.